pwa-devdocs/src/tutorials/intercept-a-target/modify-talon-results/index.md

```diff
@@ -0,0 +1,275 @@
+---
+
+In core Magento development:
+
+> A plugin, or interceptor, is a class that modifies the behavior of public class functions by intercepting a function call and running code before, after, or around that function call.
+> This allows you to substitute or extend the behavior of original, public methods for any class or interface.
+
+See: [Plugins (Interceptors)][]
+
+PWA Studio's extensibility framework provides a similar feature that allows you to intercept a talon call and surround it with custom logic.
+This is useful if you want to add tracking logic or alter the incoming or outgoing values for a talon.
+
+This tutorial teaches you how to create an extension that intercepts a talon and changes the results.
+
+## Tasks overview
+
+1. Initialize the project
+2. Create and register an intercept file
+3. Define the intercept file
+4. Make a data fetch hook
+5. Create the talon wrapper module
+6. Update the package entry point
+7. Test on a local instance
+
+## Initialize the project
+
+To intercept and wrap a talon, you need a PWA Studio extension.
+Use [`npm init`][] or [`yarn init`][] to create a new JavaScript package project for this tutorial.
+
+{: .bs-callout .bs-callout-info}
+**Note:** This is a standalone project.
+You do not need to create this inside a storefront project.
+
+Edit the `packages.json` file so it looks like the following:
+
+```json
+{
+  "name": "my-extension",
+  "version": "0.0.1",
+  "description": "A PWA Studio extension",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "index.js",
+  "license": "(OSL-3.0 OR AFL-3.0)",
+  "repository": "",
+  "dependencies": {},
+  "peerDependencies": {
+    "@magento/peregrine": "~7.0.0",
+    "@magento/pwa-buildpack": "~6.0.0",
+    "@magento/venia-ui": "~4.0.0",
+    "apollo-client": "2.6.4",
+    "graphql-tag": "~2.10.1",
+    "react": "~16.9.0",
+    "webpack": "~4.38.0"
+  },
+}
+```
+
+## Create and register the intercept file
+
+You can create the intercept file anywhere in your project.
+For this tutorial, create this file under `src/targets`.
+
+```sh
+mkdir -p src/targets && touch src/targets/my-intercept.js
+```
+
+Set the value for `pwa-studio.targets.intercept` in your project's `package.json` file to tell the build process where to find the intercept file.
+
+```diff
+    "react": "~16.9.0",
+    "webpack": "~4.38.0"
+  },
++ "pwa-studio": {
++   "targets": {
++     "intercept": "src/targets/my-intercept"
++   }
++ }
+}
+```
+
+## Define the intercept file
+
+The intercept file is where you tap into PWA Studio's extensibility framework and add your modifications.
+
+In your intercept file, add the following content:
+
+```js
+module.exports = (targets) => {
+  // Wrap the useProductFullDetail talon with this extension
+  const peregrineTargets = targets.of("@magento/peregrine");
+  const talonsTarget = peregrineTargets.talons;
+
+  talonsTarget.tap((talonWrapperConfig) => {
+    talonWrapperConfig.ProductFullDetail.useProductFullDetail.wrapWith(
```

Comment:
Might be good to call out or link to the functions available in addition to `wrapWith` (if there are any).

Reply:
there are none that I'm aware of. it seems to just have `wrapWith()`